src/rpcwallet.cpp

```diff
@@ -22,9 +22,10 @@
 
 #include "libzerocoin/Coin.h"
 #include "spork.h"
+#include "univalue/include/univalue.h"
 #include <boost/assign/list_of.hpp>
 
-#include <univalue.h>
+//#include <univalue.h>
```

Comment:
hmm, think your IDE was giving you some unique errors if was needed.

`#include <univalue.h>` should be enough.